src/Symfony/Component/Finder/Iterator/DateRangeFilterIterator.php

```diff
@@ -46,8 +46,15 @@ public function accept()
             return false;
         }
 
-        $filedate = $fileinfo->getMTime();
         foreach ($this->comparators as $compare) {
+            if ($compare->getTimeType() === $compare::TIME_TYPE_ACCESSED) {
```

Comment:
please use `DateComparator::` to access the constant rather than `$compare::`. It makes it much easier to see which constant it is.